root/edit/details/AddSeries.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {addColonText} from '../../static/scripts/common/i18n/addColon';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+
+type AddSeriesEditProps = {
+  ...EditT,
+  +display_data: {
+    +comment: string,
+    +name: string,
+    +ordering_type: SeriesOrderingTypeT,
+    +series: SeriesT,
+    +type: SeriesTypeT,
+  },
+};
+
+type Props = {
+  edit: AddSeriesEditProps,
+};
+
+const AddSeries = ({edit}: Props) => {
```

Comment:
Wouldn't it be easier to do 
```suggestion
const AddSeries = ({edit}: {edit: AddSeriesEditProps}) => {
```
here and avoid the second Props above?